utilities/post_migration.py

```
@@ -1863,6 +1900,17 @@ def check_vms(
             except Exception as exp:
                 res[vm_name].append(f"check_false_vm_power_off - {str(exp)}")
 
+    # Group 5: Plan-driven DI verification (warm vSphere)
+    if (
+        plan_resource is not None
+        and plan.get("warm_migration")
+        and source_provider.type == Provider.ProviderType.VSPHERE
+    ):
+        try:
+            _verify_warm_vsphere_di(plan_resource=plan_resource, plan=plan, di_results=di_results)
+        except Exception as exp:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)